src/main/steps/respondent/respondentcaseSequence.test.ts

```diff
@@ -118,17 +118,21 @@ describe('respondent1Sequence', () => {
 
     expect(respondentCaseSequence[28].url).toBe('/respondent/safety_concerns/domestic_abuse_risk_no');
     expect(respondentCaseSequence[28].showInSection).toBe('aboutRespondentCase');
-    expect(respondentCaseSequence[28].getNextStep({})).toBe('/respondent/task-list');
+    expect(respondentCaseSequence[28].getNextStep({})).toBe('/respondent/safety_concerns/other_concerns');
 
-    expect(respondentCaseSequence[29].url).toBe('/respondent/proceedings/start');
+    expect(respondentCaseSequence[29].url).toBe('/respondent/safety_concerns/other_concerns');
     expect(respondentCaseSequence[29].showInSection).toBe('aboutRespondentCase');
+    expect(respondentCaseSequence[29].getNextStep({})).toBe('/respondent/task-list');
 
-    expect(respondentCaseSequence[30].url).toBe('/respondent/proceedings/court-proceedings');
+    expect(respondentCaseSequence[30].url).toBe('/respondent/proceedings/start');
     expect(respondentCaseSequence[30].showInSection).toBe('aboutRespondentCase');
-    expect(respondentCaseSequence[30].getNextStep({})).toBe('/respondent/proceedings/summary');
 
-    expect(respondentCaseSequence[31].url).toBe('/respondent/proceedings/summary');
+    expect(respondentCaseSequence[31].url).toBe('/respondent/proceedings/court-proceedings');
     expect(respondentCaseSequence[31].showInSection).toBe('aboutRespondentCase');
-    expect(respondentCaseSequence[31].getNextStep({})).toBe('/respondent/task-list');
+    expect(respondentCaseSequence[31].getNextStep({})).toBe('/respondent/proceedings/summary');
```

Comment:
Hi @ShakiraKP , Please do not change anything in respondent folder. 

We already have a folder in base branch for safety concerns. Please move all your code in that folder. 
https://github.com/hmcts/prl-citizen-frontend/tree/feature/PRL-1615-c100-rebuild-journey/src/main/steps/c100-rebuild/safety-concerns

Also follow the standards which we follow for in above folders. naming convention of folder, attributes

Reply:
Changes made @yogendraU